test/common/upstream/maglev_lb_test.cc

```diff
@@ -64,13 +65,45 @@ TEST_F(MaglevLoadBalancerTest, Basic) {
   // maglev: i=5 host=127.0.0.1:90
   // maglev: i=6 host=127.0.0.1:93
   LoadBalancerPtr lb = lb_->factory()->create();
-  {
-    TestLoadBalancerContext context(0);
-    EXPECT_EQ(host_set_.hosts_[2], lb->chooseHost(&context));
+  const std::vector<uint32_t> expected_assignments{2, 4, 0, 1, 5, 0, 3};
+  for (uint32_t i = 0; i < 3 * expected_assignments.size(); ++i) {
```

Comment:
q: why 3 times the size of expected_assignments?

Reply:
Just circling around at least once; the unweighted test probes at the third iteration around.